src/Menus.ts

```diff
@@ -628,6 +639,30 @@ export class Menus<TEightBittr extends FullScreenPokemon> extends EightBittr.Com
     }
 
     /**
+     * Opens the Item menu for the item the player selected from the inventory.
+     * 
+     * @param settings   Custom attributes to apply to the menu, as well as items
+     *                   to optionally override the player's inventory.
+     */
```

Comment:
Just in case you end up not fixing it immediately - can you add a `@todo` linking to a filed issue to add the functionality?